packages/front-end/components/Metrics/MetricForm/RiskThresholds.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC } from "react";
+import { UseFormRegisterReturn } from "react-hook-form";
+
+const RiskThresholds: FC<{
+  winRisk: number;
+  winRiskRegisterField: UseFormRegisterReturn;
+  loseRisk: number;
+  loseRiskRegisterField: UseFormRegisterReturn;
+}> = ({ winRisk, winRiskRegisterField, loseRisk, loseRiskRegisterField }) => {
+  const riskError =
```

Comment:
We have an identical calculation happening in the parent index.tsx. We probably don't need to set the riskError string within the parent anymore, just use a boolean for blocking the modal's Submit.

Reply:
Ah yeah, good point. Updated